### PR TITLE
[OTE CLI] Semantic Segmentation dataset

### PR DESCRIPTION
**Pull request by Ilya-Krylov**
This PR adds semantic segmentation dataset support for OTE CLI and updates submodules to their recent state.

### CODE REVIEW

File: ote_cli/ote_cli/tools/export.py [hunk 1]
```diff
@@ -53,7 +54,7 @@ def main():
     assert args.labels is not None or args.ann_files is not None
 
     if args.labels:
-        labels = [LabelEntity(l, template.task_type) for l in args.labels]
+        labels = [LabelEntity(l, template.task_type, id=ID(f'{i}')) for i, l in enumerate(args.labels)]
```
Comment (druzhkov-paul) [inline ote_cli/ote_cli/tools/export.py:57]: Not sure how mypy treats it, but simply `id=i` should work.

Comment (Ilya-Krylov) [reply ote_cli/ote_cli/tools/export.py]: fixed

**Comment by Ilya-Krylov** [comment]
Build 18086 :clock1:

**Comment by Ilya-Krylov** [comment]
Build 18086 :heavy_check_mark:

**Comment by Ilya-Krylov** [comment]
Build 18136 :clock1:

**Comment by Ilya-Krylov** [comment]
Build 18136 :heavy_check_mark:

**Review by druzhkov-paul — APPROVED** [review]

**Pull request merged**